templates/zabbix_server.conf.erb

```diff
@@ -133,6 +133,17 @@ StartIPMIPollers=<%= @startipmipollers %>
 #
 StartPollersUnreachable=<%= @startpollersunreachable %>
 
+## Option: StartODBCPollers
```

Comment:
I'd place the `if @zabbix_version.to_f >= 6.0` before this line, as the comment doesn't make sense on 5.x either.

Reply:
Done.